### ISSUE TITLE
ResNetDUC model can not converge in the dataset CamVid?

### ISSUE THREAD

**Issue by guanfuchen** [ZijunDeng/pytorch-semantic-segmentation#25]
![image](https://user-images.githubusercontent.com/22321977/34301201-860e9954-e766-11e7-935d-2d21f89e168e.png)

First thanks for your implementing for ResNetDUC, I try to train on the dataset CamVid, but it is trained slow and seem not converge. What should I do to get a better model?

**Comment by zijundeng** [comment]
Do you check whether there is any bug, such as forgetting to call loss.backward() ?

**Comment by guanfuchen** [comment]
Sorry because it trains too slow, I thought it seems not converge.
